test/card_test.rb

```diff
@@ -11,6 +11,7 @@ def test_find_returns_one_card
       assert_equal 6, card.cmc
       assert_equal 'Sorcery — Arcane', card.type
       assert card.colors.any?{|color| color == 'Black'}
+      assert card.color_identities.any?{|color_id| color == 'B'}
```

Comment:
`color` is undefined here. Either rename the `color_id` to `color`, or vice versa.

Reply:
doh!, half awake when I wrote this for the pull this morning. Will make the changes